src/explore-education-statistics-frontend/src/components/Page.tsx

```diff
@@ -21,13 +42,22 @@ const Page = ({ children, breadcrumbs = [], wide = false }: Props) => {
         })}
       >
         <PageBanner />
-        <Breadcrumbs breadcrumbs={breadcrumbs} />
+        <Breadcrumbs
+          breadcrumbs={
+            isHomepage
+              ? undefined
+              : breadcrumbs.concat([{ name: breadcrumbLabel || title }])
+          }
+        />
 
         <main
           className="govuk-main-wrapper app-main-class"
           id="main-content"
           role="main"
         >
+          {!hideTitle && (
+            <PageTitle title={title} caption={hideCaption ? '' : caption} />
```

Comment:
`hideCaption` is redundant as `PageTitle` checks if `caption` is empty when deciding if to render the caption or not